roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -525,7 +525,7 @@ max_upload_size: "{{ matrix_synapse_max_upload_size_mb }}M"
 # an explicit url_preview_ip_range_blacklist of IPs that the spider is
 # denied from accessing.
 #
-url_preview_enabled: true
+url_preview_enabled: "{{ matrix_synapse_url_preview_enabled }}"
```

Comment:
Why add it as a string? Does Synapse understand it correctly if so?

I guess something like this would be ideal:

```
url_preview_enabled: {{ matrix_synapse_url_preview_enabled|to_json }}
```

It's what we do for other boolean variables as well.

Reply:
Was a bit too uncareful. Good find and fixed :)